client/src/state/DraftStore.ts

```diff
@@ -78,6 +79,17 @@ export const draftStore = vuexModule(
         state.cards = parsed.cards;
         state.events = events;
         state.isComplete = timelineGenerator.isDraftComplete();
+
+        const init = [] as DeckInitializer[];
```

Comment:
I think for code-splitting reasons we want the deck store to have a dependency on the draft store instead of the other way around (there might be some situations where we just need a draft store, but if we have a deck store we always need a draft store).

I'm not SUPER happy with what we're doing right now, but I'd emulate what we do for the replay store: have your base controller (Replay.vue in this case) listen for changes to the draft store and then call `.sync()` on the depending store when it sees a change.

I originally tried to do the above with dependent getters, but it just got messy.